dash-renderer/src/components/error/menu/DebugMenu.css

```
@@ -59,44 +63,114 @@
     justify-content: center;
     align-items: center;
     width: 74px;
-    margin-left: 10px;
-}
-.dash-debug-menu__button-label {
-    color: #A2B1C6;
-    font-size: 10px;
-    margin-top: 4px;
 }
 
 .dash-debug-menu__button {
-    background-color: white;
+    position: relative;
+    background-color: #B9C2CE;
     border-radius: 100%;
-    border: 1px solid #B9C2CE;
     width: 64px;
     height: 64px;
     font-size: 10px;
     display: flex;
+    flex-direction: column;
     justify-content: center;
     align-items: center;
     transition: background-color 0.2s;
-    color: black;
+    color: #fff;
+    cursor: pointer;
+}
+.dash-debug-menu__button:hover {
+    background-color: #a1a9b5;
 }
 .dash-debug-menu__button--enabled {
     background-color: #00CC96;
-    color: white;
 }
-.dash-debug-menu__button--small {
-    width: 32px;
-    height: 32px;
-    background-color: #B9C2CE;
+.dash-debug-menu__button.dash-debug-menu__button--enabled:hover {
+    background-color: #03bb8a;
 }
-.dash-debug-menu__button:hover {
+
+.dash-debug-menu__button-label {
+    cursor: inherit;
+}
+
+.dash-debug-menu__button::before {
+    visibility: hidden;
+    pointer-events: none;
+    position: absolute;
+    box-sizing: border-box;
+    bottom: 110%;
+    left: 50%;
+    margin-left: -60px;
+    padding: 7px;
+    width: 120px;
+    border-radius: 3px;
+    background-color: rgba(68,68,68,0.7);
+    color: #fff;
+    text-align: center;
+    font-size: 10px;
+    line-height: 1.2;
+}
+.dash-debug-menu__button:hover::before {
+    visibility: visible;
+}
+.dash-debug-menu__button--callbacks::before {
+    content: "Toggle Callback Graph";
+}
+.dash-debug-menu__button--errors::before {
+    content: "Toggle Errors";
+}
+.dash-debug-menu__button--available,
+.dash-debug-menu__button--available:hover {
+    background-color: #00CC96;
+    cursor: default;
+}
+.dash-debug-menu__button--available::before {
+    content: "Server Available";
+}
+.dash-debug-menu__button--unavailable,
+.dash-debug-menu__button--unavailable:hover {
+    background-color: #F1564E;
+    cursor: default;
+}
+.dash-debug-menu__button--unavailable::before {
+    content: "Server Unavailable. Check if the process has halted.";
```

```suggestion
    content: "Server Unavailable. Check if the process has halted or crashed.";
```

just in case folks wonder what "halt" means?

---

or rather, "halt" implying "stuck" / taking a long time to process vs "crashed"